src/tasks/WasmAppBuilder/PInvokeTableGenerator.cs

```diff
@@ -11,7 +11,7 @@
 using Microsoft.Build.Framework;
 using Microsoft.Build.Utilities;
 
-public class PInvokeTableGenerator : Task
+public class PInvokeTableGenerator
 {
     [Required, NotNull]
```

Comment:
Since this is no longer a task, the `Required`, and `Output` attributes should be removed.
And remove the `NotNull` ones on the properties because these are not guaranteed to be null by msbuild any longer (as not a task). Instead, these can be accepted as constructor args, and thus can be non-nullable types.